thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/diffsummary/costfunction/CostFunction.java

```diff
@@ -0,0 +1,6 @@
+package com.linkedin.thirdeye.client.diffsummary.costfunction;
```

Comment:
if this package contains different types of cost functions, the package should be plural. costfunctions or just functions

Reply:
Done!